responses/rentals/search-scope.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
If you mean attributes, then no - we use light rental serializer for search and thats all is there.